Installation/cmake/modules/UseLASLIB.cmake

```diff
@@ -1,4 +1,4 @@
 # This module setups the compiler for using LASlib library.
 # It assumes that find_package(LASLIB) was already called.
 
-add_definitions(-DCGAL_LINKED_WITH_LASLIB)
+add_definitions(-DCGAL_LINKED_WITH_LASLIB -DUSE_AS_DLL)
```

Comment:
Where does that name `USE_AS_DLL` come from? It should have a kind of namespacing: `LASLIB_USED_AS_DLL`, or 
`CGAL_USE_LASLIB_AS_DLL`...